front/src/applications/stdcm/hooks/useConflicts.ts

```diff
@@ -0,0 +1,108 @@
+import { useEffect, useState } from 'react';
+
+import type { TFunction } from 'i18next';
+
+import type { ManageTrainSchedulePathProperties } from 'applications/operationalStudies/types';
+import { processConflicts } from 'applications/stdcm/utils/fomatConflicts';
+import type { Conflict } from 'common/api/osrdEditoastApi';
+
+const useConflictsMessages = (
+  t: TFunction,
+  conflictData?: Conflict[],
+  pathProperties?: ManageTrainSchedulePathProperties
+) => {
```

Comment:
When you'll have made the change for results.input/ouput. You can pass the output directly here and modify the begining of `useEffect` (see next comment)

```suggestion
const useConflictsMessages = (
  t: TFunction,
  selectedSimulationOutput: StdcmSimulationOutputs
) => {
```